### ISSUE TITLE
buffer stream from chunks

### ISSUE THREAD

**Issue by flieks** [react-native-community/react-native-audio-toolkit#189]
Is it possible to read only new chunks of data ?
This for live uploading of the audio to my server..

Thanks

**Comment by eduardotamaki** [comment]
@flieks did you found a solution?

**Comment by flieks** [comment]
@eduardotamaki nope, i think the best solution is using webview to use the html api. 
Its the only way to not eject expo

**Comment by s4rath07** [comment]
How did you do it?
